<ui version="4.0" >
 <class>ExportXmlDialog_base</class>
 <widget class="QDialog" name="ExportXmlDialog_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Export STDF to XML</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelFileName" >
     <property name="text" >
      <string>STDF file name:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QLineEdit" name="editFileName" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer name="spacer1" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>121</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="4" >
    <layout class="QVBoxLayout" name="hBoxLayoutDialog" >
     <item>
      <widget class="QProgressBar" name="progressBar" >
       <property name="value" >
        <number>0</number>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hBoxLayoutButton" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonExport" >
         <property name="text" >
          <string>&amp;Export</string>
         </property>
         <property name="shortcut" >
          <string>Alt+E</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClose" >
         <property name="text" >
          <string>&amp;Close</string>
         </property>
         <property name="shortcut" >
          <string>Alt+C</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="3" >
    <widget class="QPushButton" name="buttonProperties" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>STDF properties</string>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="C:/Work/Dev_QT4/Galaxy_Products/gex_product/gex/sources/gex.qrc" >
       <normaloff>:/gex/icons/properties.png</normaloff>:/gex/icons/properties.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="buttonSelect" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Select the STDF file to convert</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="C:/Work/Dev_QT4/Galaxy_Products/gex_product/gex/sources/gex.qrc" >
       <normaloff>:/gex/icons/file_open.png</normaloff>:/gex/icons/file_open.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
